### ISSUE TITLE
Can I access the octave command line executable from the package?

### ISSUE THREAD

**Issue by jrminter** [flathub/org.octave.Octave#55]
I have successfully installed the Octave 5.1.0 on Ubuntu 18.04 x64. The GUI works great. Thanks for your effort.

I also use RStudio which can process code chunks in Octave in the code chunks. I need a path to a working octave command line. I can find the binaries in the application folder, but

(base) jrminter@ubuntu-18-04:~$ /var/lib/flatpak/app/org.octave.Octave/current/active/files/bin/octave-cli-5.1.0 

fails with:

/var/lib/flatpak/app/org.octave.Octave/current/active/files/bin/octave-cli-5.1.0: error while loading shared libraries: liboctinterp.so.7: cannot open shared object file: No such file or directory

Can you suggest a way to use the octave command line? What am I missing?

**Comment by mtmiller** [comment]
Yes, just use the flatpak command

    flatpak run org.octave.Octave

You can add any Octave options at the end, for example

    flatpak run org.octave.Octave --eval "ver"

You might also be interested in the simple wrappers installed in /var/lib/flatpak/exports/bin. You could add that directory to your `PATH`, and then simply use `org.octave.Octave` as the command name.

**Comment by jrminter** [comment]
Thank you so much. Your response was quite helpful.

 I set `OCTAVE_PATH` to    
"/var/lib/flatpak/exports/bin/org.octave.Octave"
in `.Renviron` and used an R chunk 

Note: I moved the Rmarkdown brackets inside the triple backticks to preserve visibility of the code in markdown here... They would normally be adjacent to the opening triple backtics

```
{r setup, include=TRUE}
knitr::opts_chunk$set(echo = TRUE)

octavePath <- Sys.getenv("OCTAVE_PATH")
octavePath
```

to retrieve the path and feed it to the Octave code chunk like this:

```
{octave, engine.path=octavePath}
   A=[1,2,3;4,5,6]
   [minval, row] = min(min(A,[],2));
   [minval, col] = min(min(A,[],1));
   [minval, row, col]
```
This chunk used two custom functions in my `~/octave` folder.
